Resources/parallax_config_clouds.toml

```diff
@@ -0,0 +1,31 @@
+# Clear to black.
+[[layers]]
+type = "clear"
+color = "#000000"
+
```

Comment:
these should be in their own folder and not just the root of resources

Reply:
Should I put them in with the parallax prototypes or stuff them somewhere else entirely?

Reply:
They have been thrown into Prototypes/Parallaxes